# ROS2 interface for DBoW2 BowVector.h
uint32 word_id
float64 word_value
